include/class.task.php

```diff
@@ -1236,7 +1236,7 @@ function update($forms, $vars, &$errors) {
             $_errors = array();
             $this->postNote(array(
                         'note' => $vars['note'],
-                        'title' => __('Task Update'),
+                        'title' => __('Task Updated'),
```

Comment:
This should be `_S()` because it is posting a permanent note to the task thread. Therefore should use the system language rather than the current user's language.